web/demo.html

```diff
@@ -1,99 +0,0 @@
-<html>
-    <head>
```

Comment:
shouldn't be removed but moved to public folder

Reply:
Done.